samples/functionapp-csharp/HelloSequence.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextImplementations;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Options;
+using Microsoft.Extensions.Configuration;
+
+namespace DurableClientSampleFunctionApp
+{
+    public class HelloSequence
```

Comment:
Let's not include any orchestrations for this app so it is clear we are using exclusively as a client.

Reply:
This app contains a function with an HTTP trigger, not an orchestrator function. I can see the confusion naming the class HelloSequence though, so I renamed it to ClientFunction. Let me know if you meant something else and I'll address that!